packages/select/src/Select.tsx

```diff
@@ -11,7 +10,7 @@ import {
   MultiValueRemove,
   DropdownIndicator,
 } from './styled';
-import { convertToOptions, findOptionByValue, themeConverter } from './utils';
```

Comment:
You didn't delete `convertToOptions `and `findOptionByValue`. Are they used somewhere else?